Content.Server/Tabletop/TabletopBackgammonSetup.cs

```diff
@@ -0,0 +1,91 @@
+using JetBrains.Annotations;
+using Robust.Shared.GameObjects;
+using Robust.Shared.Serialization.Manager.Attributes;
+
+namespace Content.Server.Tabletop
+{
+    [UsedImplicitly]
+    public class TabletopBackgammonSetup : TabletopSetup
+    {
+        [DataField("boardPrototype")]
+        public string BackgammonBoardPrototype { get; } = "BackgammonBoardTabletop";
+
+        [DataField("whitePiecePrototype")]
+        public string WhitePiecePrototype { get; } = "WhiteTabletopPiece";
+
+        [DataField("blackPiecePrototype")]
+        public string BlackPiecePrototype { get; } = "BlackTabletopPiece";
+
+
+        public override void SetupTabletop(TabletopSession session, IEntityManager entityManager)
+        {
+            var board = entityManager.SpawnEntity(BackgammonBoardPrototype, session.Position);
+
+            const float borderLengthX = 7.35f; //BORDER
+            const float borderLengthY = 5.60f; //BORDER
+
+            const float boardDistanceX = 1.25f;
+            const float pieceDistanceY = 0.80f;
+
+            float getXposition (float distanceFromSide, bool isLeftSide)
+            {
+                return isLeftSide ? -borderLengthX + (distanceFromSide * boardDistanceX)
+                : borderLengthX - (distanceFromSide * boardDistanceX);
+            }
+
+            float getYPosition (float positionNumber, bool isTop)
+            {
+                return isTop ? borderLengthY  - (pieceDistanceY * positionNumber)
+                : -borderLengthY  + (pieceDistanceY * positionNumber);
+            }
+
+            var center = session.Position;
+
+            // Top left
+            session.Entities.Add(entityManager.SpawnEntity(BlackPiecePrototype, center.Offset(getXposition(0, true), getYPosition(0, true))));
+            session.Entities.Add(entityManager.SpawnEntity(BlackPiecePrototype, center.Offset(getXposition(0, true), getYPosition(1, true))));
+            session.Entities.Add(entityManager.SpawnEntity(BlackPiecePrototype, center.Offset(getXposition(0, true), getYPosition(2, true))));
+            session.Entities.Add(entityManager.SpawnEntity(BlackPiecePrototype, center.Offset(getXposition(0, true), getYPosition(3, true))));
+            session.Entities.Add(entityManager.SpawnEntity(BlackPiecePrototype, center.Offset(getXposition(0, true), getYPosition(4, true))));
```

Comment:
I suggest putting all of these in for loops, because they all increment anyways. Apart from that, you can't really get away from "hard coding" this stuff, because thats basically just how the boardgame works.